task/update-infra-deployments/0.1/update-infra-deployments.yaml

```diff
@@ -120,7 +120,10 @@ spec:
 
         EXPIRE_MINUTES_AS_SECONDS = int(os.environ.get('GITHUBAPP_TOKEN_EXPIRATION_MINUTES', 10)) * 60
         # TODO support github enteprise
-        GITHUB_API_URL = os.environ.get('GITHUB_API_URL')
+        GITHUB_API_URL = os.environ["GITHUB_API_URL"]
```

Comment:
In case these required env variables are missing, do we want just to fail with just the KeyError?

Reply:
The error is for us only to know something is misconfigured. KeyError should be enough to let us know.